assets/sass/components/_topbar.scss

```diff
@@ -0,0 +1,141 @@
+.topbar {
+	background-color: $aplus-brand-primary;
+	border-bottom: none;
+	-webkit-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25),
+		inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+	-moz-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25),
+		inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+	box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25), inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+
+	& .navbar-brand,
+	& .navbar-nav > li > a,
+	& .navbar-toggle,
+	& .navbar-nav > .active > a,
+	& .navbar-nav > .active > a:hover {
+		@include focus-with-background-and-color;
+
+		color: #ffffff;
+		background: transparent;
+	}
+
+	& a.navbar-brand:hover,
+	& .navbar-nav > li > a:hover,
+	& .navbar-nav > .open > a,
+	& .navbar-nav > .open > a:focus,
+	& .navbar-nav > .open > a:hover,
+	.navbar-inverse .navbar-toggle:focus,
+	.navbar-inverse .navbar-toggle:hover {
+		background-color: #236aa7;
+		background-color: rgba(0, 0, 0, 0.1);
+	}
+	& .navbar-toggle .badge {
+		position: absolute;
+		right: -10px;
+		top: 4px;
+	}
+	& .navbar-header .visible-xs {
+		max-width: 250px;
+		white-space: nowrap;
+		overflow: hidden;
+		text-overflow: ellipsis;
+	}
+	& ul.visible-xs {
+		margin: 0;
+
+		& li {
+			border: 1px solid;
+			border-top: none;
+			border-color: #236aa7;
+			border-color: rgba(0, 0, 0, 0.25);
+		}
+		& li a {
+			color: white !important;
+			padding: 10px 15px !important;
+		}
+		& ul.dropdown-menu {
+			padding: 0;
+		}
+		& ul.dropdown-menu li {
+			border: none;
+			border-top: 1px solid;
+		}
+		& .header {
+			border-left: none;
+			border-right: none;
+			padding: 0.2em 1em;
+		}
+		& .header h4 {
+			font-size: 10px;
+			font-weight: bolder;
+			color: #75a5d0;
+			margin: 0;
+		}
+	}
+
+	& .navbar-nav > li.menu-groups {
+		& > a.dropdown-toggle {
+			background: rgba(255, 0, 0, 0.3);
+		}
+		& > a.dropdown-toggle:hover {
+			background: rgba(255, 0, 0, 0.4);
+		}
+		& .profile-avatars {
+			display: inline-block;
+			margin: -6px 3px;
+		}
+	}
+
+	& .navbar-nav > li.menu-groups.open > a.dropdown-toggle {
+		background: rgba(255, 0, 0, 0.4);
+	}
+
+	& .dropdown-menu > li button {
+		display: block;
+		border: none;
+		background: transparent;
+		padding: 3px 20px;
+		clear: both;
+		font-weight: 400;
+		line-height: 1.42857143;
+		color: #333;
+		white-space: nowrap;
+		text-align: left;
+		width: 100%;
+
+		&:hover {
+			background: #eee;
+		}
+	}
+
+	& .menu-notification .glyphicon {
+		margin-right: 18px;
+	}
+	& .menu-notification .badge {
```

Comment:
The original CSS did not start with `.topbar` in the rule `.menu-notification .badge`. It probably works as well, though. It is important to test these notifications since they don't appear under typical local testing. I think you can create a fake notification manually in the Django admin interface.

Reply:
It seems like I added the curly braces in the wrong place. I will review this file again.

Reply:
Actually, having these classes under the topbar may increase the specificity of the rules. But, I think I will stick to the previous sass properties.